cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/ProgramOptionConstants.java

```diff
@@ -36,4 +36,6 @@
   public static final String WORKFLOW_BATCH = "workflowBatch";
 
   public static final String SCHEDULE_NAME = "scheduleName";
+
+  public static final String CONCURRENT_PROGRAM_RUN = "concurrent.program.run";
```

Comment:
something like concurrent.runs.enabled or concurrent.runs.disabled would be better because then it is more clear what a true or false value means.